docs/getting-started-on-snowplow-open-source/setup-snowplow-on-aws/setup-destinations/setup-redshift/setup-emretlrunner/configuring-emretlrunner/index.md

```diff
@@ -1,202 +0,0 @@
----
-title: "Configure EmrEtlRunner"
```

Comment:
I suggested removing the EmrEtlRunner docs currently available in the AWS manual setup docs: https://docs.snowplow.io/docs/getting-started-on-snowplow-open-source/setup-snowplow-on-aws/setup-destinations/setup-redshift/setup-emretlrunner/ 

in favour of linking to https://docs.snowplow.io/docs/pipeline-components-and-applications/legacy/emr-etl-runner/ 

I compared section by section and they seemed to have roughly the same information. I didn't check which was most accurate by following either guide, but EmrEtlRunner is deprecated, so we're not recommending anyone follow the guide. I figured we could resurrect this version of EmrEtlRunner that was in the AWS manual setup docs from github if for some reason we needed to because some info was here, but not in the guide in the Legacy Pipeline Components section.